CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ endif()
 # Globally inherit the top-level include directory because it is difficult to
 # locate from CMakeLists.txt in source subdirectories.
 include_directories("${PROJECT_SOURCE_DIR}/include")
+file(GLOB_RECURSE chimera_headers "${PROJECT_SOURCE_DIR}/include/*.h")
```

Comment:
I would prefer not to compute these headers using a GLOB_RECURSE, because it sometimes leads to invalid caching if the files are changed.

Reply:
I wished you missed this change but failed! Maybe I'm too lazy maintaining the source list manually. :/

Let me revert this change.

Reply:
https://cmake.org/cmake/help/v3.3/command/file.html?highlight=glob#file
>Note:  We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.

Reply:
Fixed in #193 since this PR include the changes of #193.

Reply:
Yeah, I know that CMake also officially doesn't recommend that. I simply don't prefer that way. But I'm okay with not to use it though.